frontend/src/employee-frontend/components/unit/TabCalendar.tsx

```diff
@@ -69,14 +68,17 @@ const TopHorizontalLine = styled(HorizontalLine)`
   margin-top: 0;
 `
 
-const getWeekDateRange = (date: LocalDate, operationalDays: DayOfWeek[]) => {
+const getWeekDateRange = (date: LocalDate) => {
```

Comment:
Consider using existing `LocalDate` `.endOfWeek()` and `.lastDayOfMonth()` functions

Reply:
Ah, nice! fixed